lib/get-color-pair.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const colors = {
```

Comment:
I've added a comment which mentions that the colors come from o-colors --
```suggestion
// Currently hard-coding the o-colors values
// We should use o-colors values directly when that is possible.
// Currently waiting on https://github.com/Financial-Times/o-colors/issues/289
const colors = {
```